README.md

```diff
@@ -1,75 +1,176 @@
 
-```bash
-echo "https://$(oc env dc/nagios --list|grep NAGIOS_USER | awk -F'=' '{print $2}'):$(oc env dc/nagios --list|grep NAGIOS_PASSWORD | awk -F'=' '{print $2}')@$(oc get route nagios --template "{{.spec.host}}")"
-```
 
-## Building
-```shell
-$ npm install
-```
+	|  Description                               | Parameter name               | Default value           | Fail/warning/recommended             |
+	|--------------------------------------------|------------------------------|-------------------------|--------------------------------------|
+	| Min number of CPUs                         | `min_required_vCPUS`         | 2                       | fail only in strict_mode             |
+	| Min system memory per node (in MB)         | `required_mem_mb_threshold`  | 7000                    | fail only in strict_mode             |
+	| Min total free memory of all nodes (in KB) | `warning_kb_value`           | 4000000                 | warning                              |
+	| Number of PVs with 50 GB storage           | `required_50_pv`             | 0                       | warning                              |
+	| Number of PVs with 25 GB storage           | `required_25_pv`             | 1                       | warning                              |
+	| Number of PVs with  5 GB storage           | `required_5_pv`              | 0                       | warning                              |
+	| Number of PVs with  1 GB storage           | `required_1_pv`              | 1                       | warning                              |
 
-Produce a tar with the templates:
 
-```shell
-$ grunt
-```
-The produced tar will be in the ```dist``` directory.
 
-## Workflow Guidelines
+- **To deploy the MBaaS on OpenShift execute the following command**
 
-* Use the parameters section to define key values and then reference them in the rest of the template.
-* Keep version in template and package.json in sync.
+  ```bash
+  
+  cd /directory-to-fh-openshift-templates
+  # first execute the config map creation
+  oc new-app -f rhmap-mbaas-config.json
+  # create the 1 node mbaas
+  oc new-app -f fh-mbaas-template-1node.json 
 
-### Troubleshooting
+  --> Deploying template "1node-mbaas/fh-mbaas" for "fh-mbaas-template-1node.json" to project 1node-mbaas
+
+     fh-mbaas
+     ---------
+     Red Hat Mobile Backend as a Service template
+  
+
+  ``` 
+- **Monitor the deploy on OpenShift**
+
+  ```bash
+  
+  oc get pods
+  NAME                   READY     STATUS    RESTARTS   AGE
+  fh-mbaas-1-vrzfm       1/1       Running   4          7m
+  fh-messaging-1-brldn   1/1       Running   3          6m
+  fh-metrics-1-bxnwk     1/1       Running   4          7m
+  fh-statsd-1-wx4ck      1/1       Running   0          6m
+  mongodb-1-1-ff6vp      1/1       Running   0          5m
+  nagios-1-704pp         1/1       Running   0          6m
+  
+
+  ```
+
+- **Check Nagios for all health endpoints**
```

Comment:
I would include this one-liner from old version of Readme as well:
`echo "https://$(oc env dc/nagios --list|grep NAGIOS_USER | awk -F'=' '{print $2}'):$(oc env dc/nagios --list|grep NAGIOS_PASSWORD | awk -F'=' '{print $2}')@$(oc get route nagios --template "{{.spec.host}}")"`
it generates nice link, e.g: https://nagiosadmin:password@nagios-mbaas.cluster.domain.com